data/default.nix

```diff
@@ -27,9 +28,11 @@ let
         elmfile = module: "\${srcdir}/\${builtins.replaceStrings ["."] ["/"] module}.elm";
       in ''
         mkdir -p \$out/share/doc
-        \${lib.concatStrings (map (module: ''
+        \${lib.concatStrings (map (module:
+          let output = module + (if outputJavaScript then ".js" else ".html");
```

Comment:
alternatively we can just resolve `extension` in previous let block and do `\$out/\${module}\${extension}`. Not sure what is prefered.

Reply:
After thinking about it a bit I think it will be better.